tests/UI/specs/UIIntegration_spec.js

```diff
@@ -167,7 +167,11 @@ describe("UIIntegrationTest", function () { // TODO: Rename to Piwik?
 
             await page.goto("?" + urlBase + "#?" + generalParams + "&category=General_Visitors&subcategory=General_Overview&segment=" + segment);
 
-            expect(await page.screenshotSelector('.pageWrap,.top_controls')).to.matchImage('visitors_overview_segment');
+            // check that segment is selected in selector
+            const segmentTitle = await page.evaluate(() => $('.segmentationTitle').text());
+            expect(segmentTitle).to.match(/<script>_x\(\d+\)<\/script>/);
```

Comment:
This change isn't clear to me, why is this being changed

Reply:
The UI test is testing a segment. To ensure the segment is really selected the test now checks if the chosen segment is the expected one. Before that was indirectly tested by having the segment selector included in the screenshot.